package.json

```diff
@@ -379,9 +379,9 @@
     "semantic-ui-react": "2.0.3",
     "semver": "5.6.0",
     "serialize-javascript": "3.1.0",
-    "slate": "0.84.0",
-    "slate-hyperscript": "0.81.3",
-    "slate-react": "0.83.2",
+    "slate": "0.91.4",
```

Comment:
Could this affect existing sites in any way?

Reply:
@sneridagh no, I don't think so. 

The slate upgrade alone makes our tests fail, I have one PR with only that change. So, regardless of anything, if we want to make the upgrade, we need to do the test fix.